plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -2339,6 +2339,16 @@ else if (foodLevel / maxHunger < 1) {
             return null;
         });
 
+        // <--[tag]
+        // @attribute <PlayerTag.skin_model>
+        // @returns ElementTag
+        // @description
+        // Returns the type of skin model the given player has. Valid models types are: CLASSIC or SLIM.

```

Comment:
Should match the format used by other tags, something along the lines of `Returns the player's skin model, either X or Y.`